build/test_update_ext_version.py

```diff
@@ -0,0 +1,97 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+import json
+import pathlib
+import pytest
+
+from update_ext_version import main as update_version, parse_version
+
+
+@pytest.fixture
+def directory(tmpdir):
+    """Fixture to create a temp directory wrapped in a pathlib.Path object."""
+    return pathlib.Path(tmpdir)
```

Comment:
If you're purely using the fixture to construct a path to a `package.json`, then do that directly:

```suggestion
@pytest.fixture
def package_json_path(tmp_path):
    """Fixture to create a temp directory wrapped in a pathlib.Path object."""
    return tmp_path / "package.json"
```